Projects/Feature/OnboardingFeature/Sources/Onboarding/OnboardingView.swift

```diff
@@ -66,6 +66,10 @@ public struct OnboardingView: View {
     .background(Alias.Color.Background.primary)
     .setFrameMeasure(space: .global, identifier: backgroundFrame)
     .getFrameMeasure { value in
```

Comment:
음 이건 내가 봤을때 frameMeasure를 구현할때 사용한 onPreferenceChange가 인식을 못하는거 같은데
그냥 이런식으로 쓰면 아마 문제는 해결될거 같네

```suggestion
.background {
      GeometryReader { proxy in
        Color.clear
          .onChange(of: proxy.frame(in: .global)) { _, newValue in
            store.width = newValue.width
          }
      }
    }
```

Reply:
그냥 만들어둔 모디파이어 쓸 수 있는 방법 더 알아보지 말고 우선 바로 geometryReader 사용하는 방법으로 수정할까?